common/src/main/java/io/pravega/common/util/ServiceConfig.java

```diff
@@ -7,13 +7,9 @@
  *
  *     http://www.apache.org/licenses/LICENSE-2.0
  */
-package io.pravega.segmentstore.server.store;
+package io.pravega.common.util;
```

Comment:
This class absolutely does not belong in common.
common.util if for utility classes (not instantiatable, no members, only static methods) that are sufficiently generic that any application could reasonably use them. This is not a utility class and it is not common.